fili-core/src/main/java/com/yahoo/bard/webservice/web/handlers/workflow/CachingMode.java

```diff
@@ -0,0 +1,95 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web.handlers.workflow;
+
+import com.yahoo.bard.webservice.config.BardFeatureFlag;
+import com.yahoo.bard.webservice.config.SystemConfig;
+import com.yahoo.bard.webservice.config.SystemConfigProvider;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Caching config analyzer.
+ */
+public class CachingMode {
+    private static final Logger LOG = LoggerFactory.getLogger(CachingMode.class);
+
+    private static final boolean CACHE_V1 = BardFeatureFlag.DRUID_CACHE.isOn();
+    private static final boolean CACHE_V2 = BardFeatureFlag.DRUID_CACHE_V2.isOn();
+    private static final SystemConfig SYSTEM_CONFIG = SystemConfigProvider.getInstance();
+    private static final int DRUID_UNCOVERED_INTERVAL_LIMIT = SYSTEM_CONFIG.getIntProperty(
+            SYSTEM_CONFIG.getPackageVariableName("druid_uncovered_interval_limit"),
+            0
+    );
+    private static final String NO_CACHING = "No caching";
+    private static final String TTL_CACHING = "TTL caching only";
+    private static final String LOCAL_SIGNATURE_CACHING = "Local signature caching only";
+    private static final String ETAG_CACHING = "Etag caching only";
+
```

Comment:
I would like to suggest short, no spaces, and camelized names for these property values.  We may want to put more than one into a property at some point and this certainly is verbose for hand editting in a properties file.